csv2rdf/README.md

```diff
@@ -0,0 +1,24 @@
+# CSV -> RDF
+The script csv2rdf_single_subject.py converts a reconciled CSV file into a RDF turtle file.
+Steps:
+1.  Get the file that needs to be converted from the target database.
+    *   for example, in musicbrainz/csv, there is a script that converts the JSON file into a finite CSV file.
+2.  (Optional) Make that file into a CSV file and reconcile it. 
+    *   for the same example, in musicbrainz/csv/history, there are JSON files that has the reconciliation process in OpenRefine.
+3.  Move the reconciled CSV file to this current folder.
+4.  Run get_relations.py to generate a mapping.json file, fill the empty values with the references to the key properties. For example:
```

Comment:
I think this is a more complex process and needs more detail. For example, what are the 'key properties'? 

Can you give an example?

**Example**
- pass in a csv file containing reconciled data on chants with headings x, y, and z. 
- Then when you pass that file to `get_relations.py` the output will look like this....
- Then, you fill in the blanks so that it looks like this...

Reply:
Ok. I will clearly document the mapping file procedures.